### DISCUSSION TITLE
New Presentation field - Column

### DISCUSSION THREAD

**Discussion by johngrantdev** [payloadcms/payload#4752] [Feature Requests & Ideas]
I would like to be able to nest presentation columns within a row field to allow for more complex admin UI layouts. Currently this can be achieved by nesting a group within a row however this approach has an impact on the data structure.

Below is an example of the desired effect using nested groups to create multiple columns.
![layout_example](https://github.com/payloadcms/payload/assets/99842644/72035a98-1923-4930-9e27-99ae3cac758a)

I believe this could be achieved with a new presentation only column field. alternatively a generic container field might be more versatile to replace the row field eg.
```
{
    type: 'container',
    label: string | false, // optional default false
    direction: 'column' | 'row', // optional default 'column'
    fields: []
},
```

**Comment by franciscolourenco** [comment]
Came here to request something similar. Ideally it would support nesting 2 columns column inside of a row.
